docs/sources/metainfo/service.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN" "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd" [
+<!ENTITY % BOOK_ENTITIES SYSTEM "AppStream.ent">
+%BOOK_ENTITIES;
+]>
+
+<section id="sect-Metadata-Service">
+	<title>Services</title>
+
+	<section id="spec-service-introduction">
+		<title>Introduction</title>
+		<para>
+                        A service component is any software that is started and supervised by
+                        the Operating Systems "init" facility, such as systemd.
+		</para>
+		<para>
+                        A service component is any software that is started and supervised by
+                        the Operating Systems "init" facility, such as systemd.
+		</para>
+		<para>
+			The file described in this document is built upon the generic component metadata with fields specific for services (see <xref linkend="sect-Metadata-GenericComponent"/>).
```

Comment:
> Other component types also use the term "file". Maybe we can change them all in a separate PR?+

Jup, I can do that later.

> "Component" sounds wrong; we don't describe the component itself, but the format for expressing meta data about it. So what about "component metadata format"?

You are right, "component metadata format" is way better - I think just writing "metadata" is fine as well (slightly less verbose)